tests/examplefiles/nim/example.nim.output

```diff
@@ -49,14 +49,27 @@
 'search'      Name
 '\n\n'        Text
 
-'{.'          Punctuation
-'push'        Name
-' '           Text
-'callConv'    Name
-':'           Punctuation
-'cdecl'       Name
-'.'           Punctuation
-'}'           Punctuation
+'{.'          Literal.String.Other
+'p'           Literal.String.Other
+'u'           Literal.String.Other
+'s'           Literal.String.Other
+'h'           Literal.String.Other
```

Comment:
This seems like a missing `+` in a regex, those tokens should be merged together.

Reply:
Done.